### ISSUE TITLE
More status messages

### ISSUE THREAD

**Issue by Tadrix** [endless-sky/endless-sky#3418]
We now have "Ship lost! Mission failed" type messages, but could we have:
`"<ship name>" lost.`
`"<ship name>" disabled.`
`"<ship name>" overheated.`
`"<ship name>" stranded at <system>.` (when escort without ramscoop runs out of fuel in uninhabited system)
etc.

These events are rarer than usual spam, and could be useful, especially for new players who tend to keep asking questions like: "Why my ship/escort is stuck?"

**Comment by tehhowch** [comment]
You do get overheated notifications for your own ship, and you can see disabled / stranded info in the radar panel.

With #3070 you can also see anything that would manifest in the radar display in the orbits scene.

And, the player info panel happily indicates how much fuel an escort has. 
(No ship is ever truly stranded unless they're out of fuel in a system that has no solar wind (default is every system has some solar wind), and with v0.9.9 the ship will move to accelerate its fuel recharge without any player intervention.)

**Comment by Tadrix** [comment]
One more: There are messages that say that my ship is scanned, but they don't say _which_ ship is scanned. Which, if I have more than one ship makes them useless. It would be at least good to know if flagship is currently scanned.

**Comment by endless-sky** [comment]
Most of those statuses show up in the escort display, so I'm not sure it's worth adding messages for them. Especially because in some situations, you could end up with a ton of messages swamping out the more useful information.

**Comment by Tadrix** [comment]
Could we at least add scanning indication for escorts in escort display? For example, a pulsating antenna status icon next to them (and their stats) while they're scanned (and then slowly fading). A red thermometer icon while overheated, lightning icon while powerless, etc.?

**Comment by KiLEdEnNis** [comment]
ya can already see the latter 2 from the bars themselves

**Comment by Wrzlprnft** [comment]
i do think small icons would help to more or less get the entire situation at a glance. but wouldn't add them, since i am also convinced that those scale horribly with fleet size.

**Comment by Tadrix** [comment]
If some ships get stacked, don't show separate icons for them (unless all of them have certain status), but simply a warning sign (something like this, perhaps: ⚠) that something is wrong? Realistically, situational awareness for larger fleets should be worse than for small fleets anyway. If you have a huge army, you care less for individual smaller ships, than if they are all you have.

@KiLEdEnNis There's a rather big difference between "almost overheated" and "overheated".

**Comment by KiLEdEnNis** [comment]
> There's a rather big difference between "almost overheated" and "overheated".
indeed; several pixels in the bar you already see

**Comment by Laudrin** [comment]
I think "acoustical clues" / different sounds for the above mentioned and more events would be very helpful!

Especially in a fight it's hard to follow the text status messages and sometimes I miss that one of my ships was destroyed.

At the moment important messages tend to get lost under less important status messages and other ship's chatter.

**Comment by Laudrin** [comment]
Addition: Sometimes my ships are destroyed **without** a status message. It seems this is due to the ships not containing any passengers or cargo belonging to a mission.

This is intensified by the ship icon list in the corner of the screen. The idea to show the star system's name, the ship is in, is great. But that makes it even more confusing to get a quick overview over your own ships and other ships you are escorting to a destination.

My suggestion would be:
- Show the star system's name right besides the fleet icons and status, not below. That way the list won't expand and collapse all the time you visit a new star system and you could see if ships are destroyed when the column shrinks.
- A ship's loss should be worth a more intense status message, i. e. bold and red. And the player should be informed about every loss, not only mission critical ones.
- Escorted ships not belonging to the player (escort mission ships) should have a different colour or there should be a separator between those ships and your ships. At the moment, you cannot differentiate between those two types.
- And again ;-), my wish would be a sound notification if a ship gets lost.

**Comment by tehhowch** [comment]
- Feel free to provide concept mockups for the minimum game resolution (1024 x 768). The better visuals you provide, the more likely someone uses their free time to implement your idea. In my opinion, having the system on the right side of this would consume too much screen space for small and medium size screens (or large screens with a high UI zoom).
 - Feel free to provide a viable sound effect (it should meet the licensing requirements for this repo).
Note that it is unnecessary to repost 2 days later, especially if no one has posted after your previous message.
 - Most likely of your feature requests is a separator between owned and escorted ships, whether it is just spacing or an actual element remains to be seen.
 - I think getting spammed ship destruction messages is not useful - you are well aware you are taking heavy damage, unless you are simply abandoning your escorts and gallivanting around the system instead of fighting alongside / microing them.
In a different issue I mentioned that it would be beneficial for a more permanent log of "critical" events such as mission failure. I would prefer that over colored text that does not meet the game's UI scheme.

**Comment by Xanthec** [comment]
What about status messages for escort on rolling-over the escort icon?

**Comment by Amazinite** [comment]
As noted, showing too many status messages may easily result in information overload, particularly for players with large fleets, and this information can be gained from existing UI if a player needs to look for it. But I do think that ship destruction is the most important of this information, and it wouldn't hurt to provide a more obvious sign of you losing one of your ships. #8568 will cover this.
